README.md

```diff
@@ -1,8 +1,74 @@
-# api-linter-service
+# API Linting Service
 
-[![SIT](https://img.shields.io/badge/SIT-awesome-blueviolet.svg)](https://jobs.schwarz)
+## Prerequesits / general idea
 
-HTTP based Software as a Service to lint OpenApi specifications against your company API best practices using [nodeJS](https://nodejs.dev), [nestJS](https://nestjs.com/), [@stoplight/spectral-core](https://www.npmjs.com/package/@stoplight/spectral-core),[@stoplight/spectral-ruleset-migrator](https://www.npmjs.com/package/@stoplight/spectral-ruleset-migrator) and [@stoplight/spectral-rulesets](https://www.npmjs.com/package/@stoplight/spectral-rulesets)
+General idea behind this API implementation is to provide an API as a service based on the awesome [spectral schmea linter](https://github.com/stoplightio/spectral) and it´s nodeJS based SDK.
 
-Source code will be available soon...
+## Main Technologies
 
+* [Docker](https://www.docker.com/)
+* [nodeJS](https://nodejs.dev/)
+* [nestJS](https://nestjs.com/)
+* [Spectral schema linter](https://github.com/stoplightio/spectral)
+
+## Spectral API Validation
+
+The API is build on top of the [@stoplight/spectral-core](https://www.npmjs.com/package/@stoplight/spectral-core) SDK for nodeJS to execute API validations that follow the "Open API Specification" pattern. Spectral itself is Open Source and able to validate any json or yaml defined schemas with dedicated rulesets, while playing it´s strengths definitely in validation of OpenAPI Specs.
+
+## Build / Setup the local dev env
+
+First make sure to have installed the following OS dependencies for development:
+* [Docker Desktop](https://www.docker.com/products/docker-desktop)
+* [node 17.3.1](https://nodejs.org/en/blog/release/v17.3.1/)
```

Comment:
Is this dependent on the exact version, or would aiming for the last or last LTS version be sufficient?

Reply:
LTS is fine